docs/content/FormControl.mdx

```diff
@@ -0,0 +1,372 @@
+---
+componentId: form_control
+title: FormControl
+status: Alpha
+description: The FormControl component is used to render a labelled text input and, optionally, associated validation text and hint text.
+source: https://github.com/primer/react/blob/main/src/FormControl/FormControl.tsx
+storybook: '/react/storybook?path=/story/forms-inputfield--text-input-field'
+---
+
+import {FormControl, TextInputWithTokens, Autocomplete, Select, Textarea, Checkbox, Radio} from '@primer/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+
+## Examples
+
+### Basic
+
+```jsx live
+<FormControl>
+  <FormControl.Label>Name</FormControl.Label>
+  <TextInput />
+</FormControl>
+```
+
+### With complex inputs
+
+```javascript live noinline
+const DifferentInputs = () => {
+  const [tokens, setTokens] = React.useState([
+    {text: 'zero', id: 0},
+    {text: 'one', id: 1},
+    {text: 'two', id: 2}
+  ])
+  const onTokenRemove = tokenId => {
+    setTokens(tokens.filter(token => token.id !== tokenId))
+  }
+
+  return (
+    <Box display="grid" gridGap={1}>
```

Comment:
Nitpick: Can we use a gap of `3` instead of `1` in these examples? `gridGap={1}` feels a little cramped IMO


![CleanShot 2022-02-04 at 18 35 40](https://user-images.githubusercontent.com/4608155/152625884-124e7bed-15dc-47b4-85b2-dff79edba792.png)

```suggestion
    <Box display="grid" gridGap={3}>
```